### PR TITLE
ASM-7391 added resource and template for resolv.conf

### PR DESCRIPTION
**Pull request by ksw2599**

### CODE REVIEW

File: manifests/resolv.pp [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Sets the resolv.conf for DNS config
+define network::resolv (
+  $ensure,
+  $primaryDns = undef,
```
Comment (ripienaar) [inline manifests/resolv.pp]: snake case please

**Comment by ksw2599** [comment]
@ripienaar does this look good now? please review and let me know if I need to modify anything else. Thanks.

**Comment by ripienaar** [comment]
Yes looks good. I guess it needs merging with others but +1 from me

File: manifests/resolv.pp [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Sets the resolv.conf for DNS config
+define network::resolv (
+  $ensure,
+  $primary_dns = undef,
+  $secondary_dns = undef,
+  $domain = undef,
+) {
+  file { "/etc/resolv.conf":
+    ensure  => "${ensure}",
```
Comment (ripienaar) [inline manifests/resolv.pp]: General observation but no need to quote this. Just `ensure => $ensure` Is fine and avoid needless interpolation. Not needed to fix for merge just general comment.

**Pull request merged**